app/jobs/process_delius_data_job.rb

```diff
@@ -5,18 +5,16 @@ class ProcessDeliusDataJob < ApplicationJob
 
   include ApplicationHelper
 
+  CommunityDeliusData = Struct.new(:noms_no, :tier, :team_code, :crn,
+                                   :service_provider, :ldu_code, :mappa, :mappa_levels, :offender_manager, :unallocated?)
+
   def perform(nomis_offender_id)
-    DeliusData.transaction do
+    ApplicationRecord.transaction do
       logger.info("[DELIUS] Processing delius data for #{nomis_offender_id}")
 
       DeliusImportError.where(nomis_offender_id: nomis_offender_id).destroy_all
-      delius_data = DeliusData.where(noms_no: nomis_offender_id)
-      if delius_data.count > 1
-        DeliusImportError.create! nomis_offender_id: nomis_offender_id,
-                                  error_type: DeliusImportError::DUPLICATE_NOMIS_ID
-      elsif delius_data.count == 1
-        import_data(delius_data.first)
-      end
+
+      import_data(OpenStruct.new(OffenderService.get_community_data(nomis_offender_id)))
```

Comment:
Should this `OpenStruct` actually be a `CommunityDeliusData`?

Otherwise I can't find a use of the `CommunityDeliusData` struct.